autogpt_platform/backend/backend/data/model.py

```diff
@@ -402,3 +402,96 @@ class RefundRequest(BaseModel):
     status: str
     created_at: datetime
     updated_at: datetime
+
+
+class NodeExecutionStats(BaseModel):
+    """Execution statistics for a node execution."""
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    error: Optional[Exception] = None
+    walltime: float = 0
+    cputime: float = 0
+    cost: float = 0
+    input_size: int = 0
+    output_size: int = 0
+    llm_call_count: int = 0
+    llm_retry_count: int = 0
+    input_token_count: int = 0
+    output_token_count: int = 0
+
+
+class NodeExecutionStatsStrError(BaseModel):
+    """Execution statistics for a node execution."""
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    error: Optional[str] = None
+    walltime: float = 0
+    cputime: float = 0
+    cost: float = 0
+    input_size: int = 0
+    output_size: int = 0
+    llm_call_count: int = 0
+    llm_retry_count: int = 0
+    input_token_count: int = 0
+    output_token_count: int = 0
+
+
+def convert_node_execution_stats(
```

Comment:
nit: instead of creating this generic function out of context, why don't you put this as a class method and use self, instead of stats. e.g: convert_to_str_err or something